roles/config-docker/README.md

```diff
@@ -1,25 +1,25 @@
-Set of Roles
-============
+config-docker
```

Comment:
This change was already merged as part of PR #436 ... can you please rebase this new PR to ensure all the existing stuff has been picked up?